keps/sig-auth/4193-tokenrequest-requester-info-enhancements/README.md

```diff
@@ -0,0 +1,616 @@
+* `serviceaccount_bound_tokens_issued_total{bound_object_kind="Node"}` - new metric that is incremented whenever a bound token is issued that references a Node (explicitly added)
+* `serviceaccount_bound_tokens_issued_with_identifier_total` - new metric that is incremented whenever a token that contains an identifier/JTI is issued
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Additionally, the metrics detailed above provide a clear signal as to whether these features are being used.
+
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+
+###### How can someone using this feature know that it is working for their instance?
+
+For the node info part, using the TokenRequest API and inspecting the contents of the issued JWTs for a token bound to a Pod.
+For JTIs, using the TokenRequest API and then inspecting the contents of the issued JWT for any ServiceAccount token.
+
+For the validation/verification, the user can use the SelfSubjectAccessReview API to check whether the token is still valid.
+To do so, they'd need to obtain a token that is bound to a Pod, delete the corresponding Node object that the Pod is scheduled
+on, and observe that the token is no longer valid via the SelfSubjectAccessReview API.
+
+A similar process could be used for tokens bound to Node objects directly.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+N/A
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+No
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+
+### Dependencies
+
+None
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No
+
+### Scalability
+
+N/A
+
+###### Will enabling / using this feature result in any new API calls?
+
+No
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+Additional audit log annotation keys, as well as extending the JWT claims we embed into service account tokens.
+
+The maximum size of a UUID is X bytes.
+The maximum size of a Node object's name is Y bytes.
+The maximum size of a Node object's UID is Z bytes.
```

Comment:
were these supposed to be filled in? The generated UUID for JTI and node default to 36 bytes, and a node names have a max length of 253 bytes.